example/src/XmtpContext.tsx

```diff
@@ -9,7 +9,7 @@ const XmtpContext = createContext<{
   setClient: () => {},
 });
 export const useXmtp = () => useContext(XmtpContext);
-export function XmtpContextProvider({ children }) {
+export const XmtpContextProvider: FC<any> = ({ children }) => {
```

Comment:
i think `FC<PropsWithChildren>` would work better here

Reply:
ahhh yes, I missed that..